### DISCUSSION TITLE
Help needed for implementing Deepgram Flask Tutorial

### DISCUSSION THREAD

**Discussion by ali-rafiei** [deepgram/community#329] [General help] [answered]
Hi,

I've been trying to implement Live Stream Speaker Diarization into a Flask web application by [following this tutorial](https://deepgram.com/learn/live-transcription-flask), and my team and I have been having trouble implementing it into our own Flask web app. For reference, our flask web app is very similar to the [following tutorial guide](https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-i-hello-world) and the code is [here](https://github.com/miguelgrinberg/microblog/tree/v0.1).

Our team thinks that it's maybe to do with the tutorial running the program as "python main.py" whereas ours runs as "flask run", and that the code in the tutorial isn't quite compatible with a full flask web app (some errors with the deepgram-sdk). We've been looking into converting the tutorial to use websockets to make Deepgram API requests but not quite sure how to. 

We've been at a loss on how to implement this and any help would be greatly appreciated. If possible, we would love to have a quick 15 minute meeting or so to demonstrate our troubles more efficiently.

Thank you!
Ali

**Comment by jjmaldonis** [comment, marked as answer]
Hey @ali-rafiei, happy to chat. You can send me an email at jason dot maldonis at deepgram dot com and we'll find a time that works.
